practiceapp/src/main/java/com/practiceapp/practiceapp/entity/CommunityEntity.java

```diff
@@ -63,14 +63,6 @@
 
     // References to users and posts:
```

Comment:
I think this comment refers to deleted getters. Just to prevent confusion we may also delete this comment.

Reply:
Actually, I wrote it to make it clear that we are holding references rather than objects. But if it's confusing, I can delete it.

Reply:
Sorry, I got it now. Since we will not see the deleted functions, there will not be such confusion in future :) It can stay. Thank you.